test/integration/features/basic.feature

```diff
@@ -139,62 +139,33 @@ Feature: Basic
   Scenario: User can login to the server
```

Comment:
Right, the actual scenario title is wrong. I was misled by the comment above which is the actual scenario here. "The user can deploy an application". Note, that this was the encompassing scenario and needed to be marked with a comment since it stretched over multiple scenarios. This was a sign for me that we are really dealing with a single scenario here. 

Also, imo the current scenario is much more concise and has less repetition. 12 lines for the whole things. I can take this in in one go. Previously I could not even see the whole thing w/o scrolling.

>  but let's say we can try it. 

Let's do that. You can always change it back. I think we agree that there is too much repetition in the current tests and often things are stated which are not relevant in a given context (a lot probably due to cut&paste). This is an attempt to have fewer scenarios with less repetition. I don't see a problem that a high-level scenario consists of multiple minishift/oc commands. I often have the feeling that there is a one scenario one minishift/oc command thinking which is imo wrong. 

Obviously, there is a balance somewhere and taste plays a big role as well here. In this context, I would love to hear the opinion of someone like Gojko Adzic at some stage.